concent_api/api-integration-force-get-task-result-test.py

```diff
@@ -42,7 +42,7 @@ def upload_new_file_on_cluster(task_id = '0', part_id = '0', current_time = 0):
     file_content    = task_id
     file_size       = len(file_content)
     file_check_sum  = 'sha1:' + hashlib.sha1(file_content.encode()).hexdigest()
-    file_path       = '{}/{}/result'.format(task_id, part_id)
+    file_path       = 'blender/result/{}/{}.{}.zip'.format(task_id, task_id, part_id)
```

Comment:
Please rename `part_id` to `subtask_id` because it's supposed to be the subtask ID.

If there's any other place in code that calls it `path_id`, please find it and rename it too.

Reply:
done :+1: